portfolio/src/main/webapp/index.html

```diff
@@ -40,7 +40,10 @@ <h3 id="text">This is my portfolio! I will present some of my most important asp
         <div id="language-container"></div>
       </div>
       <!-- Container used to display the returned greeting from showContent() -->
```

Comment:
Make sure to update the comment if this is no longer the case (now that you have updated the functionality).

Reply:
Done!